core/client/fs/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -157,6 +159,7 @@ public BlockInStream getInStream(long blockId, Protocol.OpenUfsBlockOptions open
       WorkerNetAddress workerNetAddress = location.getWorkerAddress();
```

Comment:
@yupeng9  I think we'd better to put this for loop into the else branch, due to if `(blockInfo.getLocations().isEmpty()` false, this for make sense. What do you think?